src/modules/timeTracker/timeTracker.redux.js

```diff
@@ -0,0 +1,39 @@
+import { createActions, createReducer } from 'reduxsauce';
+import Immutable from 'seamless-immutable';
+
+export const { Types: TimeTrackerTypes, Creators: TimeTrackerActions } = createActions(
+  {
+    start: null,
+    stop: null,
+    addSample: null,
+    saveResult: ['result'],
+  },
+  { prefix: 'TIME_TRACKER_' }
```

Comment:
```suggestion
  { prefix: 'TIME_TRACKER/' }
```
Zmienialiśmy chyba to w boilerplate. :)
Generalnie dobrze dać tego slasha, bo wtedy ładniej widać "domenę". Twórcy libek też tak robią i dlatego my też idziemy w tym kierunku.